soh/soh/Enhancements/randomizer/location_access/dungeons/gerudo_training_ground.cpp

```diff
@@ -230,10 +230,7 @@ void RegionTable_Init_GerudoTrainingGround() {
         //Events
         EventAccess(&logic->MQGTGRightSideSwitch,    []{return logic->CanUse(RG_MEGATON_HAMMER);}),
         EventAccess(&logic->GTGPlatformSilverRupees, []{return logic->CanUse(RG_FIRE_ARROWS) && logic->CanUse(RG_HOVER_BOOTS);}),
-    }, {
-        //Locations
-        LOCATION(RC_GERUDO_TRAINING_GROUND_MQ_DINOLFOS_CHEST, logic->IsAdult && (logic->CanUse(RG_MASTER_SWORD) || logic->CanUse(RG_KOKIRI_SWORD) || logic->CanUse(RG_BIGGORON_SWORD))),
```

Comment:
removed this since it appears lower down with better logic & I'm not sure how this chest is supposed to be at this location